pandas/core/indexes/multi.py

```diff
@@ -981,7 +951,8 @@ def copy(
         """
         name = kwargs.get("name")
         names = self._validate_names(name=name, names=names, deep=deep)
-
+        if "labels" in kwargs:
+            raise TypeError("'labels' argument has been removed; use 'codes' instead")
```

Comment:
same

Reply:
ideally i think we should just get kwargs out of the signature of this function, but im fine with adding a test for this in the interim